### DISCUSSION TITLE
pybullet renders wrong color

### DISCUSSION THREAD

**Discussion by OhmySallyGarden** [bulletphysics/bullet3#3606] [General]
I load the URDF model of kinova jaco, and I find that the color of the robot in the simulation is not corresponding to the camera image. 
![wrong_color](https://user-images.githubusercontent.com/37887274/56413379-e57a9a80-62b9-11e9-9cf3-c69f0a64fd3e.png)
upper left of the image and the environment are not same.
The urdf description is : 

[urdf.txt](https://github.com/bulletphysics/bullet3/files/3097432/urdf.txt)

**Comment by OhmySallyGarden** [comment]
The upper left of the image is what I want the physics engine to render. And I do not know how  to use  getVisualShapeData() and changeVisualShape() to only change the ring of the arm rather than the whole link.

**Comment by madurner** [comment]
Hey, could you share any code to be able to help you
